application/controllers/HostController.php

```diff
@@ -184,6 +184,15 @@ public function servicesAction()
         $limitControl = $this->createLimitControl();
         $paginationControl = $this->createPaginationControl($services);
         $viewModeSwitcher = $this->createViewModeSwitcher($paginationControl, $limitControl);
+        $sortControl = $this->createSortControl(
+            $services,
+            [
+                'service.display_name, host.display_name' => t('Name'),
```

Comment:
Remove `host.display_name`. It's redundant since all the services belong to the same host.